elastic/kibana
# [Infra] Add alerts section to the container views overview

Closes #185757

## Summary

This PR adds the alerts section to the container views overview.

- Docker Container

![image](https://github.com/elastic/kibana/assets/14139027/91bf9feb-e664-477a-9086-c9063ff034ae)


- Non-docker container

![image](https://github.com/elastic/kibana/assets/14139027/369a920e-8e41-4c38-9b9c-9c3e2107963d)



## Testing
- Now the oblt cluster has Docker containers so we can use it for testing
- Go to Inventory and select "Docker Containers"
- Click on a Docker Container and check the alerts section
   - If there are no active alerts the section is closed
   - The "Create Rule" button should be visible  
   - Create a rule and check the alerts using the "show all" button
   - Return to Inventory and open the container again
- Click on a non-docker container
   - Alerts section should be visible
   - The "Create Rule" button should not be visible  


https://github.com/elastic/kibana/assets/14139027/747ccaee-598f-40f9-8464-ea297c6193a7
